data/formats-data.js

```diff
@@ -5052,7 +5059,7 @@ exports.BattleFormatsData = {
 		tier: "UU",
 	},
 	cubchoo: {
-		randomBattleMoves: ["icebeam", "surf", "hiddenpowergrass", "superpower"],
+		randomBattleMoves: ["blizzard", "surf", "playrough", "superpower", "hail"],
```

Comment:
It gets ice punch as well.

Reply:
but giving it ice punch makes non-hail sets possible